src/design-system/styles/template/custom/code.njk

```diff
@@ -0,0 +1,124 @@
+---
+title: Block areas – Template
+---
+
+{# Example that changes every setting in the template #}
+
+{% from "back-link/macro.njk" import govukBackLink %}
+{% from "skip-link/macro.njk" import govukSkipLink %}
+{% from "header/macro.njk" import govukHeader %}
+{% from "phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "footer/macro.njk" import govukFooter %}
+
+{% extends "template.njk" %}
+
+{% set htmlClasses = 'app-html-class' %}
+{% set htmlLang = 'fr' %}
+{% set assetPath = '' %}
+{% set themeColor = 'blue' %}
+{% set bodyClasses = 'app-body-class' %}
+
+{% block pageTitle %}GOV.UK - Le meilleur endroit pour trouver des services gouvernementaux et de l'information{% endblock %}
+
+{% block headIcons %}
+    {{ super() }}
+{% endblock %}
+
+{% block head %}
+  <!--[if !IE 8]><!-->
+    <link href="/govuk-frontend/all.css" rel="stylesheet" />
+  <!--<![endif]-->
+
+  {# For Internet Explorer 8, you need to compile specific stylesheet #}
+  {# see https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/supporting-internet-explorer-8.md #}
+  <!--[if IE 8]>
+    <link href="/govuk-frontend/all-ie8.css" rel="stylesheet" />
+  <![endif]-->
+
+  {# For older browsers to allow them to recognise HTML5 elements such as `<header>` #}
+  <!--[if lt IE 9]>
+    <script src="/html5-shiv/html5shiv.js"></script>
+  <![endif]-->
+{% endblock %}
+
+{% block bodyStart %}
+{% endblock %}
+
+{% block skipLink %}
+  {{ govukSkipLink({
+    href: '#main-content',
+    text: 'Passer au contenu principal'
+  }) }}
+{% endblock %}
+
+{% block header %}
+  {{ govukHeader({
+    homepageUrl: "#",
+    containerClasses: "govuk-width-container",
+    serviceName: "Nom du service",
+    serviceUrl: "#",
+    navigation: [
+      {
+        href: '#1',
+        text: 'Élément de navigation 1',
+        active: true
+      },
+      {
+        href: '#2',
+        text: 'Élément de navigation 2'
+      },
+      {
+        href: '#3',
+        text: 'Élément de navigation 3'
+      }
+    ]
+  }) }}
+{% endblock %}
+
+{% set mainClasses = 'app-main-class' %}
+
+{% block main %}
+  {{ super() }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "alpha"
+    },
+    html: 'C\'est un nouveau service - vos <a class="govuk-link" href="#">commentaires</a> nous aideront à l\'améliorer.'
+  }) }}
+  {{ govukBackLink({
+    "href": "#",
+    "text": "Retourner"
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">Bonjour tous le monde.</h1>
```

Comment:
If we update the previous example, ensure this is changed. 
Also a suggestion could we have the example in Welsh?

Reply:
Yeah Welsh would make more sense, I'll try and find a Welsh speaker.

Reply:
Could we use the translated page from a live service? At least for the majority of the example content